nengo/neurons.py

```diff
@@ -260,6 +311,14 @@ def gain_bias(self, max_rates, intercepts):
         bias = 1 - gain * intercepts
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the inverse of gain_bias."""
+        intercepts = (1 - bias) / gain
+        max_rates = 1.0 / (self.tau_ref - self.tau_rc * np.log1p(
+            1.0 / (gain * (intercepts - 1) - 1)))
+        max_rates = np.nan_to_num(max_rates)
```

Comment:
I have to think more about these things, but at the moment it seems to me:
* Intercepts should never be >= 1 (*) or if they are it should be treated as an error (not sure if that means an exception or non-finite values). Compare #1248, #1212, #1231, #1243 (especially [this comment](https://github.com/nengo/nengo/pull/1243#issuecomment-267787806)).
* The gain should never be <= 0 or if it is it should also be treated as an error. Potentially I could see returning max_rates of 0 for a gain of 0, but keep the intercepts as `nan` (because there isn't a defined intercepts and returning 0 seems to me misleading, while `nan` at least indicate that something's weird). Or maybe an exception is the better solution.

(*) This only applies to LIF neurons (not sigmoid neurons), but this function only applies to LIF neurons.